docs/essentials/organizations.md

```diff
@@ -3,10 +3,16 @@ title: Organizations
 sidebar_position: 1
 ---
 
-Organizations are the top level of the Account hierarchy within DevCycle, and are where all Users and Projects are managed. Users within an organization will have the same permissions across all projects, and a user may be part of multiple Organizations.
+Organizations are the top level of the Account hierarchy within DevCycle, and are where all Users and Projects are managed. Users within an organization will have the same [permissions](/extras/permissions) across all projects, and a user may be part of multiple Organizations.
 
 To view the current Organization (or change Organizations) click on your user avatar on the top right of the DevCycle Dashboard. The active Organization will have a green badge next to it.
 
+:::tip
+
+If [granular permissions](/extras/permissions) have been activated in your Organization, you will have more customization options for Project permissions.
```

Comment:
there's only really one level of permissions. I'd probably cut this info box, or re-word it as "permissions is only available on business or enterprise plans, if you'd like to turn on permissions message us at support@" or something like that.

Reply:
Serene was okay with this as well so updating now.